waterloop-site/src/components/SustainableTech/FAQ/FAQ.tsx

```diff
@@ -1,5 +1,33 @@
-import React from 'react';
+import React from "react";
+import Collapsible from "./collapsible";
```

Comment:
We generally try to separate dependency imports and custom imports at the top of the file by a newline. Can you move the `Collapsible` and `Content` (custom) imports after the `React` and `styled` (dependency) imports, and separate the two groups by a newline?